libs/math/include/math/tensor.hpp

```diff
@@ -210,7 +210,6 @@ class Tensor
 
   bool Resize(SizeVector const &shape, bool copy = false);
   bool Reshape(SizeVector const &shape);
-  bool ResizeFromShape(SizeVector const &shape);
```

Comment:
@baykaner will probably have an opinion about removing this

Reply:
@baykaner for the avoidance of doubt, this is a declaration without a definition - this function cannot be called atm.